data/json/item_groups.json

```diff
@@ -4405,6 +4537,35 @@
     ]
   },{
     "type" : "item_group",
+    "id" : "plumber_gear",
```

Comment:
This one is too good: toolbox, bio_toolbox, bio_flashlight, useful tools - all with good chances.